File: Jody/app/src/main/java/com/infor/jody/model/Driver.java
package com.infor.jody.model;

import com.orm.SugarRecord;

/**
 * Created by jbarron on 3/15/2016.
 */
public class Driver extends SugarRecord{

    public int driverId;
    public String fname;
    public String lname;
    public String mname;
    public String schedule;
    public String mobile;

    public Driver(){

    }

    public Driver(int driverId, String fname, String lname, String mname, String schedule, String mobile) {
        this.driverId = driverId;
        this.fname = fname;
        this.lname = lname;
        this.mname = mname;
        this.schedule = schedule;
        this.mobile = mobile;
    }

    @Override
    public String toString() {
        return this.lname + ',' + this.fname;
    }
}

File: Jody-Web/models/Driver.php
<?php

namespace app\models;

use Yii;

/**
 * This is the model class for table "driver".
 *
 * @property integer $driver_id
 * @property string $fname
 * @property string $lname
 * @property string $mname
 * @property string $schedule
 * @property string $mobile
 *
 * @property Trip[] $trips
 */
class Driver extends \yii\db\ActiveRecord
{
    /**
     * @inheritdoc
     */
    public static function tableName()
    {
        return 'driver';
    }

    /**
     * @inheritdoc
     */
    public function rules()
    {
        return [
            [['fname', 'lname', 'mobile'], 'required'],
            [['fname', 'lname', 'mname'], 'string', 'max' => 100],
            [['schedule'], 'string', 'max' => 50],
            [['mobile'], 'string', 'max' => 20]
        ];
    }

    /**
     * @inheritdoc
     */
    public function attributeLabels()
    {
        return [
            'driver_id' => 'Driver ID',
            'fname' => 'Fname',
            'lname' => 'Lname',
            'mname' => 'Mname',
            'schedule' => 'Schedule',
            'mobile' => 'Mobile',
        ];
    }

    /**
     * @return \yii\db\ActiveQuery
     */
    public function getTrips()
    {
        return $this->hasMany(Trip::className(), ['driver_id' => 'driver_id']);
    }

    /**
     * @inheritdoc
     * @return DriverQuery the active query used by this AR class.
     */
    public static function find()
    {
        return new DriverQuery(get_called_class());
    }
}
